data/human/human missions.txt

```diff
@@ -11259,3 +11259,217 @@ mission "Patch Stone Waiting"
+			label enter
+			`	In single file, the three of you enter the hallway behind the opening. Strangely, the stone walls are perfectly smooth, and the light that guides you through seems to come out of nowhere. "To use alien technology for something like this," Sophie laughs, "that's totally our dad."`
+			`	For a long while the three of you continue through the stone hallway, the light following your movements, until it opens up into a dome-shaped room brilliantly bathed in blue light. On the walls, there are all manner of alien artifacts lined up next to each other, but at the center of the room there is a pedestal with a sword inserted into it. A small tablet stands in a socket in front of it.`
+			choice
+				`	(Go up to the sword.)`
+					goto "sword scene"
+				`	(Take a look around first.)`
+			`	You walk around to the right of the dome to the collection of alien objects, all steel in color and rather blocky. "Wow, he must have always brought us back the small items from his expeditions, because these artifacts over here are huge!" Liam exclaims from another part of the dome. "How did he even move all these here?"`
+			`	Now that Liam points it out, many of the larger artifacts over his way look similar to the Sheragi technology you studied with Albert Foster.`
+				to display
+					has "Sheragi Archaeology: The Emerald Sword 3: done"
+			`	After some time, Sophie calls you both to come and look at the tablet.`
+			choice
+				`	(Go over to Sophie.)`
+			label "sword scene"
+			`	A short message, written on the tablet in green ink, appears as you approach:`
+			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. It was my life's work.`
+			`			"But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
+			`			"This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
+			`			"Your father."`
+			`	Understanding the personal nature of the message, you leave the two alone for a bit, walking around and looking at the artifacts on the left of the dome.`
+			`	After some time, the siblings call you back. Sophie says, "Captain <last>, Liam and I have thought about this, and we think that we would never be able to live up to our father's expectations if we kept this sword. We can study all these artifacts and understand more about the universe through them, but we cannot study this one. To understand its power, it needs to be used.`
```

Comment:
What would a spaceship captain do with a sword? Even if it's super lightweight, a melee weapon isn't going to do much against enemies with long-range firepower, ie 99% of the enemies the player encounters outside of ship combat. How would first last even know how to use a sword?